tests/test_integration.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from dotenv import find_dotenv, load_dotenv
+
+import app
+from flask_config import Config
+
+
+@pytest.fixture
+def client():
+    # Use our test integration config instead of the 'real' version
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+    # Create the new app.
+    test_app = app.create_app()
+    # Use the app to create a test_client that can be used in our tests.
+    with test_app.test_client() as client:
+        yield client
+
+
+def test_index_page(requests_mock, client):
```

Comment:
`requests_mock` is not patched and therefore does not resolve (i.e. the test cannot be run)

This error blocks this review being passed.

Reply:
updated requirements file which should have `request_mock` dependency